### PR TITLE
Add final non-hint text to app

### PR DESCRIPTION
**Pull request by alexelash**
## Overview
We received final copy from the client—this PR adds it in everywhere except for in the hint text (which will be covered in #96).

Connects #131 

### Demo
Screen Shot 2019-08-15 at 11 43 08 AM
Screen Shot 2019-08-15 at 11 44 02 AM

### Notes
- While I was in there, we added paragraph breaks to long chunks of copy within the Meet the Fish page
- @dboyer I would like to push back on two pieces of copy shown below (client-supplied text is overtop screenshots in pink):
**Test your skills page**
I think we can go a more concise route—the "and/" is not really needed, as people will assume that they could do both.
Screen Shot 2019-08-15 at 9 57 46 AM
**Screensaver**
Similar issue here—the suggested button text is pretty long for an action button. Short and sweet would be better.
Screen Shot 2019-08-15 at 11 25 34 AM

## Testing Instructions
* `git pull`
* Go through text against the client-supplied document

### CODE REVIEW

**Comment by fungjj92** [comment]
I'd even suggest "jump in" for the screensaver button. I just don't associate diving with a river, more a pool, lake, or maybe an ocean

**Comment by fungjj92** [comment]
Or "wade in" which is very river-associative but less fun

**Comment by alexelash** [comment]
@fungjj92 I don't mind "dive" personally. I think it gets across water + action without much thinking required.

File: src/app/src/components/AboutSlide.js [hunk 1]
```diff
@@ -100,12 +100,14 @@ export default class AboutSlide extends Component {
             descriptionIntro,
             description,
             job,
+            formerEmployee,
```
Comment (fungjj92) [inline src/app/src/components/AboutSlide.js]: This totally works, a nitpick of naming clarity -- can this be `isFormerEmployee` ? Which suggests a boolean prop-type

File: src/app/src/components/MeetTheFishModal.js [hunk 2]
```diff
@@ -333,6 +335,37 @@ export default class MeetTheFishModal extends Component {
             );
         });
 
+        const overviewParagraphs = fish.overview.map((paragraph, index) => (
```
Comment (fungjj92) [inline src/app/src/components/MeetTheFishModal.js:338]: This works, but as clean up we can create these paragraphs more iteratively. I think this is working code, but might be some kinks and it's definitely not properly formatted.

```
const fishKeys = ['overview', 'characteristics', 'habitat', 'lifespan'];
const paragraphs = fishKeys.map(key => {
  return fish[key] && fish[key].map((paragraph, index) => (
    <section id={'section-' + key}>
        <StyledHeading as='h4' variant='small'>
             {key} //Titlecase this
         </StyledHeading>
            <StyledText as='p' key={fish.commonName + key + index}>
                {paragraph}
            </StyledText>
    </section>
  ));
});
```

File: src/app/src/util/constants.js [hunk 3]
```diff
@@ -368,293 +369,532 @@ export const FISH_CATALOG = [
...
-        characteristics:
-            'Flathead catfish have the scaleless, strong body and the well- developed pectoral and dorsal fin spines typical of catfish. The tail is only slightly indented, or may appear square or rounded. The dorsal fin is high, and the lower jaw projects past the upper jaw. The body looks long and slender. The upper portion of the flathead catfish’s body is yellowish brown to dark, even purplish brown, with black or brown mottling on lighter brown sides. The belly is grayish or yellowish white. It does have a flat-looking head, very wide and depressed. The chin barbels are white to yellow, the fins are mottled, and the anal fin, which has fewer than 16 rays, is short and rounded. Except f
```
Comment (fungjj92) [inline src/app/src/util/constants.js]: typo ==> `environment`

**Review by fungjj92 — APPROVED** [review]
The paragraphs are a big improvement to the modals! I just left some clean up comments, one copy edit, but this is working as is. So close to the finish line!!

File: src/app/src/components/MeetTheFishModal.js (hunk 2 above)
Comment (alexelash) [reply src/app/src/components/MeetTheFishModal.js:338]: I love this and it basically worked, but the `StyledHeading` was showing up for each new paragraph:
Screen Shot 2019-08-20 at 12 47 26 PM

For time, I'm going to leave as is.

File: src/app/src/components/AboutSlide.js (hunk 1 above)
Comment (alexelash) [reply src/app/src/components/AboutSlide.js]: Changed, good call.

File: src/app/src/components/MeetTheFishModal.js (hunk 2 above)
Comment (fungjj92) [reply src/app/src/components/MeetTheFishModal.js:338]: Ugh this is my bad -- I fed you bad code. The styled heading should've been placed a level up.

```
const fishKeys = ['overview', 'characteristics', 'habitat', 'lifespan'];
const paragraphs = fishKeys.map(key => {
  if (fish[key]) {
    const subParagraphs = fish[key].map((paragraph, index) => (
        <section id={'section-' + key}>
            <StyledText as='p' key={fish.commonName + key + index}>
                {paragraph}
            </StyledText>
        </section>
      ));
    return (
       <>
        <StyledHeading as='h4' variant='small'>
             {key} //Titlecase this
         </StyledHeading>
         {subParagraphs}
      </>
    );
  }
});
```

**Pull request merged**